### ISSUE TITLE
[BUG] epicli fails on task "repository : Wait for yum lock to be released" on CentOS Minimal

### ISSUE THREAD

**Issue by to-bar** [epiphany-platform/epiphany#2067]
**Describe the bug**
epicli apply fails with error:

```
TASK [repository : Wait for yum lock to be released] ***************************
ERROR cli.engine.ansible.AnsibleCommand - fatal: [vm1.localdomain]: FAILED! => {"changed": false, "msg": "repoquery is required to use list= with this module. Please install the yum-utils package."} 
```

**How to reproduce**
1. Prepare VM using CentOS-7-x86_64-Minimal-1810.iso
2. Run `epicli apply` (provider: any)

**Environment**
- Cloud provider: None
- OS: CentOS 7.6 minimal

**Additional context**
Issue introduced recently in #1879.

**Comment by przemyslavic** [comment]
Actually this is for `RedHat` `Azure`, not `CentOS`.
@to-bar restoring cron job doesn't work at all.
The command `/var/tmp/rhui-update-client mv /etc/cron.daily/rhui-update-client` does not appear to be correct 😏
Moving this task back to TODO.

**Comment by to-bar** [comment]
@przemyslavic great shot! :dart:
Fixed in #2095.

**Comment by przemyslavic** [comment]
:heavy_check_mark: Fixed. Now restoring `rhui-update-client` works properly.
